### PR TITLE
FLUID-4051: Uploader : Added message to use a browser that supports HTML5

### PR DESCRIPTION
**Pull request by sakshamsaxena**
Updated the HTML files to include the message which would be
appropriately displayed.

### CODE REVIEW

**Comment by jobara** [comment]
@sakshamsaxena thank you for your pull request. We ask that all our contributors sign a CLA. Could you please fill out the Fluid CLA and send it to me. https://wiki.fluidproject.org/display/fluid/Fluid+Licensing You can ping me in the IRC channel (Justin_o) if you have more questions on that.

For background reading, here's a good overview of what CLAs are and why they're important: http://oss-watch.ac.uk/resources/cla

**Comment by jobara** [comment]
@sakshamsaxena thanks for sending me the CLA

File: src/components/uploader/css/Uploader.css [hunk 1]
```diff
@@ -485,6 +485,10 @@ html>body .fl-uploader span.fl-uploader-browse {
     display:block;
 }
 
+.fl-uploader-no-js-note {
```
Comment (jobara) [inline src/components/uploader/css/Uploader.css]: I think this class should probably be renamed. 

* Look at the [naming conventions](https://wiki.fluidproject.org/display/docs/FSS+Naming+Conventions). I think what you have here is quite close to the naming structure.
* The name seems specific to the case where JS isn't enabled, but also is used when HTML5 features aren't present. This should have a more generic name like `fl-uploader-basic-note` or `fl-uploader-basic-message`

File: demos/uploader/index.html [hunk 2]
```diff
@@ -52,11 +52,12 @@
     </head>
     <body>
         <div class="flc-overviewPanel fl-overviewPanel-container"></div>
-        <!-- Basic upload controls, used when JavaScript is unavailable -->
+        <!-- Basic upload controls, used when JavaScript/HTML5 is unavailable -->
         <form method="post" enctype="multipart/form-data" class="fl-progEnhance-basic">
             <p>Use the Browse button to add a file, and the Save button to upload it.</p>
             <input name="fileData" type="file" />
             <input class="fl-uploader-basic-save" type="submit" value="Save"/>
+            <p class="fl-uploader-no-js-note">NOTE : You are using an outdated version of browser. Please upgrade your browser which supports HTML5 <noscript>and JavaScript </noscript>features.</p>
```
Comment (jobara) [inline demos/uploader/index.html]: Maybe a message more like "To use the full featured Uploader, please use a browser that supports HTML5 and JavaScript". @colinbdclark what do you think?

Comment (sakshamsaxena) [inline demos/uploader/index.html]: True, that did come across my mind, but then, wouldn't it be simply better if the user _does_ upgrade it for once and all? :sweat_smile: Let me know what you think

File: src/components/uploader/css/Uploader.css (hunk 1 above)
Comment (sakshamsaxena) [inline src/components/uploader/css/Uploader.css]: The expanse of your wiki is quite daunting to look at first :sweat_smile: I'll change this in a while.

File: demos/uploader/index.html (hunk 2 above)
Comment (jobara) [inline demos/uploader/index.html]: Well it's up to the user, they may have specific reasons why they do not want to or cannot upgrade their browser. Or they may prefer to navigate sites without scripts running all the time.

**Comment by sakshamsaxena** [comment]
Hey. Did some changes like you said. I hope they're okay now.

File: demos/uploader/index.html [hunk 3]
```diff
@@ -52,11 +52,12 @@
     </head>
     <body>
         <div class="flc-overviewPanel fl-overviewPanel-container"></div>
-        <!-- Basic upload controls, used when JavaScript is unavailable -->
+        <!-- Basic upload controls, used when JavaScript/HTML5 is unavailable -->
         <form method="post" enctype="multipart/form-data" class="fl-progEnhance-basic">
             <p>Use the Browse button to add a file, and the Save button to upload it.</p>
             <input name="fileData" type="file" />
             <input class="fl-uploader-basic-save" type="submit" value="Save"/>
+            <p class="fl-uploader-no-js-note">NOTE : To use the full featured Uploader, please use a browser that supports HTML5 <noscript>and JavaScript </noscript>features.</p>
```
Comment (jobara) [inline demos/uploader/index.html]: The class `fl-uploader-no-js-note` needs to be updated to match `fl-uploader-basic-note` which is in the css.

Comment (jobara) [inline demos/uploader/index.html]: You don't need the extra space between "Note" and ":". You could write "Note: ".

File: src/components/uploader/css/Uploader.css [hunk 4]
```diff
@@ -485,6 +485,10 @@ html>body .fl-uploader span.fl-uploader-browse {
     display:block;
 }
 
+.fl-uploader-basic-note {
+    font-weight: bold;
+}
+
```
Comment (jobara) [inline src/components/uploader/css/Uploader.css]: If the styling is only going to make the text bold, maybe use a `<strong>` element around the text instead.

File: demos/uploader/index.html (hunk 3 above)
Comment (jobara) [inline demos/uploader/index.html]: The above comments also apply to changes to the Uploader.html file

**Comment by jobara** [comment]
@sakshamsaxena are you still working on this pull request?

**Pull request closed without merging**